<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_srcImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bitwiseImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hLinesImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vLinesImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_src">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>原图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bitwise">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>二值化图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hLines">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>水平直线</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vLines">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直直线</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bitwiseVLineImg">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bitwiseVLine">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>二值化垂直线</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_edgesImg">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_edges">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>边缘图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dilateEdges">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>膨胀后的边缘图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dilateEdgesImg">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_resultImg">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>512</width>
      <height>67</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_result">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结果图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
